pkg/apiserver/user/auth.go

```diff
@@ -35,6 +35,7 @@ import (
 var (
 	ErrNS                  = errorx.NewNamespace("error.api.user")
 	ErrUnsupportedAuthType = ErrNS.NewType("unsupported_auth_type")
+	ErrLackPrivileges      = ErrNS.NewType("lack_required_privileges")
```

Comment:
How about calling it insufficient privileges? As our privilege rule is complicated and some privileges are required only in some cases.

```suggestion
	ErrInsufficientPriv = ErrNS.NewType("insufficient_privileges")
```